avaframe/ana5Utils/distanceTimeAnalysis.py

```diff
@@ -0,0 +1,653 @@
+""" functions to convert to a different coordinate system and
+    produce a range-time diagram from simulation results
+    options: 1) range-time diagram from radar's field of view capped to this
+             2) thalweg-time diagram from beta point of view along avalanche path
+"""
+
+
+import numpy as np
+import matplotlib as mpl
+from matplotlib import pyplot as plt
+import logging
+import pathlib
+
+# Local imports
+from avaframe.in3Utils import cfgUtils
+from avaframe.in3Utils import logUtils
+import avaframe.in2Trans.ascUtils as IOf
+import avaframe.out3Plot.outTopo as oP
+import avaframe.in3Utils.geoTrans as gT
+import avaframe.out3Plot.outDistanceTimeAnalysis as dtAnaPlots
+import avaframe.ana3AIMEC.aimecTools as aT
+import avaframe.out3Plot.plotUtils as pU
+import avaframe.com1DFA.com1DFA as com1DFA
+import avaframe.in3Utils.fileHandlerUtils as fU
+
+# create local logger
+# change log level in calling module to DEBUG to see log messages
+log = logging.getLogger(__name__)
+
+
+def getRadarLocation(cfg):
```

Comment:
is def splitIniValueToArraySteps(cfgValues, returnList=False) not doing exactly this?

Reply:
no, this function makes a 1D vector but I need here a 2x2 array